addons/point_of_sale/static/src/js/Screens/ProductScreen/ProductScreen.js

```diff
@@ -27,6 +27,7 @@ odoo.define('point_of_sale.ProductScreen', function(require) {
                 client: this._barcodeClientAction,
                 discount: this._barcodeDiscountAction,
                 error: this._barcodeErrorAction,
+                multiple: this._barcodeMultipleAction,
```

Comment:
I chose `multiple`, because it can also be used for the other type of barcodes. Also, as I did implementation inside the `point_of_sale`, I didn't want to mention the gs1.